### PR TITLE
Null color picker

### PR DESCRIPTION
**Pull request by KatrinaTurner**
Allows you to choose what color is used to fill buckets in the heatmap that have NO data as opposed to data values of 0.

### CODE REVIEW

File: src/module.ts [hunk 1]
```diff
@@ -97,6 +97,11 @@ export const plugin = getPanelPluginOrFallback(
             showIf: (config: HeatmapFieldConfig) =>
               config.colorPalette !== 'custom' && config.colorPalette !== 'fieldOptions',
           })
+          .addColorPicker({
+            path: 'nullValueColor',
+            name: 'Null Value Color',
```
Comment (marcusolsson) [inline src/module.ts:102]:
```suggestion
            name: 'Null value color',
```

For consistency; Other options are in sentence case. Also, I'm wondering if the name alone is clear enough or whether it warrants a description. We could also wait to collect feedback.

File: src/module.ts [hunk 2]
```diff
@@ -97,6 +97,11 @@ export const plugin = getPanelPluginOrFallback(
             showIf: (config: HeatmapFieldConfig) =>
               config.colorPalette !== 'custom' && config.colorPalette !== 'fieldOptions',
           })
+          .addColorPicker({
```
Comment (marcusolsson) [inline src/module.ts:100]: We need to check in which version of Grafana the color picker option was added. Currently, the plugin can run on >=7.0.3, but we might need to bump the required minimum required version depending on when this was introduced.

**Review by marcusolsson — COMMENTED** [review]
Nice! Looks really clean 👍 

I'll make sure to test this during the weekend so we can get this merged! In the meantime, would you mind adding docs for this in the README?

Also, did you make any changes to the .gitignore? Seems to be only whitespace changes? If this was unintentional, could you revert it?

**Comment by KatrinaTurner** [comment]
Thank you!  I added a description into the README and reverted the .gitignore changes, they were unintentional.

**Review by marcusolsson — CHANGES_REQUESTED** [review]
I've tested this and it seems to work great even on 7.0.3, so no version bump needed! 🎉 

It seems like your editor might change the whitespace in the edited files (like the README.md and gitignore). Are you perhaps using Windows? I'm not entirely sure, but your git client might be set up to [commit Windows-style line endings](https://stackoverflow.com/questions/10418975/how-to-change-line-ending-settings).

I recommend that you look into it to make it easier to work on cross-platform projects. But if you don't have the time to look into it right now, you could revert the README.md changes and I'll add your docs update in a separate commit.

**Comment by KatrinaTurner** [comment]
Hi! Yes I am on Windows, I forgot to change the settings for the line endings.  Sorry about that!  I just fixed it and pushed.
Yay! Glad it works on 7.0.3! :)

**Review by marcusolsson — APPROVED** [review]
Thank you for this! 🎉

**Pull request merged**